app/uk/gov/hmrc/essttpstubs/services/EligibilityService.scala

```diff
@@ -37,7 +37,7 @@ class EligibilityService @Inject() (eligibilityRepo: EligibilityRepo)(implicit e
   }
 
   def eligibilityData(eligibilityRequest: EligibilityRequest): Future[Option[EligibilityCheckResult]] =
-    eligibilityRepo.findEligibilityDataByTaxRef(eligibilityRequest.idValue)
+    eligibilityRepo.findEligibilityDataByTaxRef(eligibilityRequest.identification.headOption.map(_.idValue.value).getOrElse(""))
```

Comment:
so we're less  reliant on order perhaps it'd be safer to cycle through all the id's and return the first id that has a response stored for it. We should be careful about doing something like `.getOrElse("")` since that default case is almost certainly something which is never gonna map to anything meaningful in this case.